src/platform/utilities/feature-toggles/feature-toggle-query-list.json

```diff
@@ -1,3 +1,6 @@
 {
-    "featureToggleQueryList": ["facilityLocatorShowCommunityCares"]
-}
\ No newline at end of file
+  "featureToggleQueryList": [
+    "facilityLocatorShowCommunityCares",
+    "profileShowDirectDeposit"
```

Comment:
we should move these to constants somewhere so they can also be used in the selectors.

Reply:
made a ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/1935